app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
   </head>
 
   <body>
```

Comment:
- [ ] Kindly, please add these lines of codes to the layout to allow easy log out after the user has logged in.
![log_out](https://user-images.githubusercontent.com/105460631/218879896-e2773b2b-2ea8-450f-a6ff-ed187d28280f.PNG)